src/current/cockroachcloud/cloud-api.md

```diff
@@ -408,7 +504,7 @@ Where:
 
   - `{cluster_id}` is the unique ID of this cluster.
   {{site.data.alerts.callout_info}}
-  The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-a-basic-cluster.md %}).
+  The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-your-cluster.md %}).
```

Comment:
This heading and following are actually missing the `raw` section. Another vote for taking it out everywhere.

Reply:
Saving this as a follow-up action